pandas/core/strings.py

```diff
@@ -1095,24 +1095,48 @@ def str_pad(arr, width, side='left', fillchar=' '):
 
 def str_split(arr, pat=None, n=None):
     """
+    Split strings around given separator/delimiter.
+
     Split each string (a la re.split) in the Series/Index by given
     pattern, propagating NA values. Equivalent to :meth:`str.split`.
```

Comment:
Add a reference to this in a See Also section

Reply:
reference to re.split?

Reply:
Sorry you can ignore this. I was thinking of linking to `str.split` since you mention it as equivalent here, but it's the same exact docstring as noted in other comments